### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by andreasb242**

### CODE REVIEW

File: src/control/xml/XmlPointNode.cpp [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #include "XmlPointNode.h"
 
-XmlPointNode::XmlPointNode(const char* tag) : XmlNode(tag)
+XmlPointNode::XmlPointNode(const char* tag)
+ : XmlNode(tag),
+   points(NULL),
+   timestamp(0)
```
Comment (morrolinux) [inline src/control/xml/XmlPointNode.cpp]: is it like setting timestamp=0 ? I've never seen this before

**Comment by morrolinux** [comment]
Damn now I see... I really don't get it. I'm using vscode which (apparently) changed tabs as well. (shouldn't it be 4 spaces?)

File: src/control/xml/XmlPointNode.cpp (hunk 1 above)
Comment (andreasb242) [reply src/control/xml/XmlPointNode.cpp]: Yes, this is the standard C++ way, I didn't even know that setting in the header file does work at all, seems to be a new C++11 feature (or a G++ Extension)

**Comment by andreasb242** [comment]
You should use Tabs (size = 4 Spaces), Not everything was from you, I also changed small things which was already there.

**Pull request merged**